encryption/rollo/rollo-I/param/rollo-I-128.yaml

```diff
@@ -0,0 +1,13 @@
+name: ROLLO-I-128
+failure probability: -28
+failure probability comment: 2^-28
+sizes:
+  pk: 696
+  sk: 40
+  ct|sig: 696
+  comment: 'SessionKey: 64'
```

Comment:
this is `msg`

Reply:
About ROLLO III : "**Updates for April 22nd, 2020**
• For sake of simplicity we now only consider the ROLLO-I and ROLLO-II schemes"

Reply:
https://pqc-rollo.org/doc/rollo-specification_2020-04-21.pdf, page 3